scrapers/items/bills.py

```diff
@@ -15,3 +16,11 @@ class BillStub:
     identifier: str
     session: str
     chamber: Chamber
+
+
+@dataclass
+class BillItem:
```

Comment:
What do you see as the benefit of having one class called `BillItem` that has properties for entities that are not bills? I think in my head I would imagine `BillItem` would be the equivalent of `Bill`, there would be a `VoteEventItem` that would be equivalent to `VoteEvent`, etc. etc... 

Is it not possible to have a full-fledged class work as a Scrapy Item class? If not, and scrapy just needs a dataclass that wraps around each one, I'd still want to separate things out, as it just reads as confusing to me to have a `BillItem` that is a wrapper around a `VoteEvent` sometimes, a `State` sometimes, a `Bill` sometimes, etc..

Reply:
@jessemortenson Scrapy should be return dict, None or scrapy item. Bill or State can't be an scrapy item. While exporting the data, we need to use some methods (pre_save, jurisdictions ...) of Bill or State so couldn't use such exporting method like bill.as_dict().

 I've updated the item classes as you mentioned. Please review it and let me know it's agreeable.
Thanks.